beesdoo_base/views/partner.xml

```diff
@@ -1,43 +1,74 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record model="ir.ui.view" id="beesdoo_partner_form_view">
-        <field name="name">beesdoo.partner.form.view</field>
-        <field name="model">res.partner</field>
-        <field name="inherit_id"  ref="point_of_sale.view_partner_property_form" />
-        <field name="arch" type="xml" >
-            <field name="name" position="replace">
-                <field name="name" class="oe_read_only" />
-                <field name="first_name" placeholder="First Name"  class="oe_edit_only" attrs="{'invisible' : [('company_type', '=', 'company')]}"/>
-                <field name="last_name" placeholder="Last Name"  class="oe_edit_only" default_focus="1"/>
-            </field>
-            <field name="website" position="after">
-                <field name="eater" attrs="{'invisible': [('company_type', '=', 'company')]}"/>
-                <field name="parent_eater_id" attrs="{'invisible' : [('eater', '!=', 'eater')]}" />
-            </field>
-            <xpath expr="//notebook" position="inside">
-                <page string="Carte de Membre"
-                    attrs="{'invisible': ['|', ('customer', '=', False), ('eater', '!=', 'worker_eater')]}">
-                    <separator string="Mangeurs" />
-                    <field name="child_eater_ids" widget="many2many_tags" />
-                    <separator string="Cartes" />
-                    <field string="Cartes" name="member_card_ids">
-                        <tree editable="bottom">
-                            <field name="barcode" />
-                            <field name="activation_date" />
-                            <field name="end_date" />
-                            <field name="responsible_id" />
-                            <field name="comment" />
-                        </tree>
-                    </field>
-                </page>
-            </xpath>
-            <field name="barcode" position="attributes">
-                <attribute name="attrs">{'invisible' : [('eater', '!=', 'worker_eater')]}</attribute>
-            </field>
-            <field name="barcode" position="after">
-                <field name="parent_barcode" attrs="{'invisible' : [('eater', '!=', 'eater')]}" />
-            </field>
-        </field>
-    </record>
+	<record id="MemberCard Wizard" model="ir.ui.view">
+		<field name="name">New MemberCard Wizard</field>
```

Comment:
Change the view to adapt to text fields
<separator string="Reason" />
                <field name="new_comment" string="Raison" editable="True" />
                <field name="partner_id" invisible="1" />
                <footer>
                    <button type="object" name="create_new_card" string="Créer"
                        class="oe_highlight" />
                    <button special="cancel" string="Annuler" />
                </footer>